### DISCUSSION TITLE
NextJS api scheduling and cancelling

### DISCUSSION THREAD

**Discussion by yonatanlavy** [vercel/next.js#64555] [Help]
### Summary

I'm looking to make an app that can schedule and cancel if needed a scheduled api.
For example, I want to post a Facebook post in 2 days, and tomorrow I want to edit that post to change to a new one(for example by changing the schedule time for 3 days from now). Is there a way I can do that in Nextjs?

**Comment by elie222** [comment]
Store the time in the database. Then call your API when the time runs out. You could use something like Qstash to schedule the API call.

You could also set up a Node.js server to do this with timeouts.
